src/main/scala/ai/privado/utility/Utilities.scala

```diff
@@ -395,6 +396,12 @@ object Utilities {
     }
   }
 
+  def getAPIIdentifierFromCode(codeSnippet: String, variablePattern: String): Option[String] = {
```

Comment:
@dbMundada Can we have some test case around this method?

Reply:
added tests